bioinflib.py

```diff
@@ -0,0 +1,141 @@
+#! /usr/bin/env python
+
+from typing import List, Tuple, Union, Dict
+from bioinflib_folder_1 import RNA_DNA_Module as RD
+from bioinflib_folder_1 import FILTER_Module as FILTER
```

Comment:
Зачем тут эта строчка и откуда ты её взял?:)
```suggestion
from typing import List, Tuple, Union, Dict
from bioinflib_folder_1 import RNA_DNA_Module as RD
from bioinflib_folder_1 import FILTER_Module as FILTER
```

Reply:
Если ответишь как оно называется и зачем нужно - не буду снижать оценку

Reply:
Actually I don't know ахха. Когда мы делали задание-калькулятор, мне сказали, что эта строчка позволяет запускать код из командной строки, типа так происходит распознавание языка программирования (??)

Reply:
кто-то из ассистентов?

Reply:
Тиммейт)

Reply:
пойду снижу оценку тиммейту

Reply:
Или мне за то, что не чекаю информацию дважды)

Reply:
Хорошо, дваждый тебе и еще тиммейту